rust/Cargo.lock.in

```diff
@@ -1250,3 +1252,24 @@ dependencies = [
  "thiserror",
  "time",
 ]
+
+[[package]]
+name = "zerocopy"
+version = "0.7.35"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "1b9b4fd18abc82b8136838da5d50bae7bdea537c574d8dc1a34ed098d6c166f0"
+dependencies = [
+ "byteorder",
+ "zerocopy-derive",
+]
+
+[[package]]
+name = "zerocopy-derive"
+version = "0.7.35"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "fa4f8080344d4671fb4e831a13ad1e68092748387dfc4f55e356242fae12ce3e"
+dependencies = [
+ "proc-macro2 1.0.86",
+ "quote 1.0.37",
```

Comment:
This commit has a lot more than i intended. Shouldn't have done this, I think. @jasonish when are we supposed to upgrade the lock file?

Reply:
The lock file will get updated every time we add or change a dependency.  In master I think its OK to let the whole lock file update, if it were on a release branch I'd be more fine grained about it.

- the lock file changes should be in the same commit as the Cargo.toml changes
- maybe a reason why for the upgrade in the commit or the issue?

Reply:
ok. Thank you very much!